blog/_posts/btcpay-server-1-12-0.md

```diff
@@ -0,0 +1,132 @@
+
+In the next major release 1.13.0, we plan to move out any remaining altcoin code into independent plugins, moving them completely out of the main repository. 1.12.0 ensures proper foundations are in place to achieve that. 
+
+### End of Support for Postgresql 11
+
+As Postgresql 11 has reached its end-of-life (EOL) status, we will no longer provide support for it in BTCPay Server. While it should still work, we recommend considering an upgrade to a more recent version of Postgresql for ongoing compatibility. Learn more about the EOL status of Postgresql [here](https://www.postgresql.org/support/versioning/.)
+
+
+## 🎉 New features
+
+### 🛍️ Point of Sale improvements
+
+Based on extensive [research](https://blog.btcpayserver.org/case-study-hodlhodl-2023/) and feedback we continuously receive from people around the world onboarding retailers, we’re bringing in several important UX/UI enhancements in 1.12.0.
+
+![BTCPay Server POS](/images/1-12/1.12.0-Keypad-Plus.gif)
+
+We’ve added the ability to add up amounts through **+ functionality**, making it easier to calculate the total, and **clear** out wrongly added amounts. 
+
+Cashiers can now see a **transaction history** of the Point of sale app, ensuring easier invoice troubleshooting and refunding for users with permission to do so. We also improved **error messaging** on the checkout when using NFC cards and **optimized receipts printing**.
+
+![BTCPay Server POS Transacitons Histor](/images/1-12/1.12.0-Keypad-Transaction-History.gif)
+
+In case of [too many categories](https://blog.btcpayserver.org/btcpay-server-1-11-0/) in the POS cart view, categories are now scrollable. Starting 1.12.0, categories and filters can now be hidden from the main view, but they’re on by default if you’re using categories.
+
+
+![BTCPay Server POS Categories](/images/1-12/274262116-43fb3c0a-7cde-45a9-9928-a4ccfc2d45bc.gif)
+
+**Item editor** used in both Point of Sale and Crowdfunding apps got improved, allowing easier adding, drag and drop and editing experience for products.
+
+![BTCPay Server POS Item Editor](/images/1-12/1.12.0-Keypad-Editr.gif)
+
+### ⚡ Generate BoltCards at scale through BTCPay directly!
+
+Inspired by the success of circular economies using BTCPay on physical POS devices and NFC cards, we’ve added direct integration of [BoltCard](https://www.boltcard.org) into our Pull Payment system, allowing anyone to issue physical NFC cards. We demonstrated the setup during our BTCPay Day in [Tokyo](https://x.com/BtcpayServer/status/1720707779680464954?s=20), and replicated it during the [AfroBitcoin](https://www.afrobitcoin.org) conference for merchants.
+
+Quite simply, any BTCPay user now has the ability to not only generate NFC cards for spending for their local economies and merchants, but also have an easy way to do that at scale, thanks to our BTCPay Vault, it takes only a few seconds. The cards generated aren’t static. 
```

Comment:
not sure what "The cards generated aren't static." means here. You mean that they can be reprogrammed over and over or is it because of the top-up feature;  afaik both is true for any boltcard though?

Reply:
Good point, removed here https://github.com/btcpayserver/blog/pull/45/commits/9ff989b9c4298b826497907f1df8238ed18b969d